icons/bow-arrow.svg

```diff
@@ -0,0 +1,17 @@
+<svg
+  xmlns="http://www.w3.org/2000/svg"
+  width="24"
+  height="24"
+  viewBox="0 0 24 24"
+  fill="none"
+  stroke="currentColor"
+  stroke-width="2"
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+  <path d="M12.92 5.423A13 13 0 0 0 2 5l5 5" />
+  <path d="M17 3h4v4" />
+  <path d="M18.575 11.076a13 13 0 0 1 .428 10.925L14 17" />
+  <path d="M7 14a1.707 1.707 0 0 0-1.207.5l-2.646 2.646A.5.5 0 0 0 3.5 18H6v2.5a.5.5 0 0 0 .854.354L9.5 18.207A1.707 1.707 0 0 0 10 17v-2a1 1 0 0 0-1-1z" />
+  <path d="M9.707 14.293 21 3" />
```

Comment:
```suggestion
  <path d="M12.92 5.423a13 13 0 0 0-9.424-.951 1 1 0 0 0-.374 1.65L7 10" />
  <path d="M17 3h4v4" />
  <path d="M18.575 11.076a13 13 0 0 1 .955 9.428 1 1 0 0 1-1.65.375L14 17" />
  <path d="M7 14a1.7 1.7 0 0 0-1.207.5l-2.646 2.646A.5.5 0 0 0 3.5 18H5a1 1 0 0 1 1 1v1.5a.5.5 0 0 0 .854.354L9.5 18.207A1.7 1.7 0 0 0 10 17v-2a1 1 0 0 0-1-1z" />
  <path d="M9.707 14.293 21 3" />
```

<a title="Open lucide studio" target="_blank" href="https://lucide-studio.vercel.app/edit?value=%3Cpath+d=%22M12.92+5.423a13+13+0+0+0-9.424-.951+1+1+0+0+0-.374+1.65L7+10%22+/%3E%3Cpath+d=%22M17+3h4v4%22+/%3E%3Cpath+d=%22M18.575+11.076a13+13+0+0+1+.955+9.428+1+1+0+0+1-1.65.375L14+17%22+/%3E%3Cpath+d=%22M7+14a1.7+1.7+0+0+0-1.207.5l-2.646+2.646A.5.5+0+0+0+3.5+18H5a1+1+0+0+1+1+1v1.5a.5.5+0+0+0+.854.354L9.5+18.207A1.7+1.7+0+0+0+10+17v-2a1+1+0+0+0-1-1z%22+/%3E%3Cpath+d=%22M9.707+14.293+21+3%22+/%3E"><img alt="icons" width="200px" src="https://lucide.dev/api/gh-icon/PHN2ZwogIHhtbG5zPSJodHRwOi8vd3d3LnczLm9yZy8yMDAwL3N2ZyIKICB3aWR0aD0iMjQiCiAgaGVpZ2h0PSIyNCIKICB2aWV3Qm94PSIwIDAgMjQgMjQiCiAgZmlsbD0ibm9uZSIKICBzdHJva2U9ImN1cnJlbnRDb2xvciIKICBzdHJva2Utd2lkdGg9IjIiCiAgc3Ryb2tlLWxpbmVjYXA9InJvdW5kIgogIHN0cm9rZS1saW5lam9pbj0icm91bmQiCj4KICA8cGF0aCBkPSJNMTIuOTIgNS40MjNhMTMgMTMgMCAwIDAtOS40MjQtLjk1MSAxIDEgMCAwIDAtLjM3NCAxLjY1TDcgMTAiIC8+CiAgPHBhdGggZD0iTTE3IDNoNHY0IiAvPgogIDxwYXRoIGQ9Ik0xOC41NzUgMTEuMDc2YTEzIDEzIDAgMCAxIC45NTUgOS40MjggMSAxIDAgMCAxLTEuNjUuMzc1TDE0IDE3IiAvPgogIDxwYXRoIGQ9Ik03IDE0YTEuNyAxLjcgMCAwIDAtMS4yMDcuNWwtMi42NDYgMi42NDZBLjUuNSAwIDAgMCAzLjUgMThINWExIDEgMCAwIDEgMSAxdjEuNWEuNS41IDAgMCAwIC44NTQuMzU0TDkuNSAxOC4yMDdBMS43IDEuNyAwIDAgMCAxMCAxN3YtMmExIDEgMCAwIDAtMS0xeiIgLz4KICA8cGF0aCBkPSJNOS43MDcgMTQuMjkzIDIxIDMiIC8+Cjwvc3ZnPg==.svg"/><br/>Open lucide studio</a>

Reply:
Nice. Hadn't thought to round the ends of the bow because I think of them as pointy too. One potential modification, further to yours: if the radius of those curves is increased to $4-2\sqrt{2}$ = 1.17px the arcs line up with the pixel grid.

I'm happy with either. :-)

<a title="Open lucide studio" target="_blank" href="https://lucide-studio.vercel.app/edit?value=%3Cpath+d=%22M17+3h4v4%22+/%3E%3Cpath+d=%22M18.575+11.082a13+13+0+0+1+1.048+9.027+1.17+1.17+0+0+1-1.914.597L14+17%22+/%3E%3Cpath+d=%22M7+10+3.29+6.29a1.17+1.17+0+0+1+.6-1.91+13+13+0+0+1+9.03+1.05%22+/%3E%3Cpath+d=%22M7+14a1.7+1.7+0+0+0-1.207.5l-2.646+2.646A.5.5+0+0+0+3.5+18H6v2.5a.5.5+0+0+0+.854.354L9.5+18.207A1.7+1.7+0+0+0+10+17v-2a1+1+0+0+0-1-1z%22+/%3E%3Cpath+d=%22M9.707+14.293+21+3%22+/%3E"><img alt="icons" width="200px" src="https://lucide.dev/api/gh-icon/PHN2ZwogIHhtbG5zPSJodHRwOi8vd3d3LnczLm9yZy8yMDAwL3N2ZyIKICB3aWR0aD0iMjQiCiAgaGVpZ2h0PSIyNCIKICB2aWV3Qm94PSIwIDAgMjQgMjQiCiAgZmlsbD0ibm9uZSIKICBzdHJva2U9ImN1cnJlbnRDb2xvciIKICBzdHJva2Utd2lkdGg9IjIiCiAgc3Ryb2tlLWxpbmVjYXA9InJvdW5kIgogIHN0cm9rZS1saW5lam9pbj0icm91bmQiCj4KICA8cGF0aCBkPSJNMTcgM2g0djQiIC8+CiAgPHBhdGggZD0iTTE4LjU3NSAxMS4wODJhMTMgMTMgMCAwIDEgMS4wNDggOS4wMjcgMS4xNyAxLjE3IDAgMCAxLTEuOTE0LjU5N0wxNCAxNyIgLz4KICA8cGF0aCBkPSJNNyAxMCAzLjI5IDYuMjlhMS4xNyAxLjE3IDAgMCAxIC42LTEuOTEgMTMgMTMgMCAwIDEgOS4wMyAxLjA1IiAvPgogIDxwYXRoIGQ9Ik03IDE0YTEuNyAxLjcgMCAwIDAtMS4yMDcuNWwtMi42NDYgMi42NDZBLjUuNSAwIDAgMCAzLjUgMThINnYyLjVhLjUuNSAwIDAgMCAuODU0LjM1NEw5LjUgMTguMjA3QTEuNyAxLjcgMCAwIDAgMTAgMTd2LTJhMSAxIDAgMCAwLTEtMXoiIC8+CiAgPHBhdGggZD0iTTkuNzA3IDE0LjI5MyAyMSAzIiAvPgo8L3N2Zz4=.svg"/><br/>Open lucide studio</a>

Reply:
> Nice. Hadn't thought to round the ends of the bow because I think of them as pointy too. One potential modification, further to yours: if the radius of those curves is increased to 4 − 2 2 = 1.17px the arcs line up with the pixel grid.
> 
> I'm happy with either. :-)
> 
> [<img alt="icons" width="200px" src="https://camo.githubusercontent.com/881edd92355ce030a0bbf4b18dcca5280d00324869fc3cbdb0e196c6008f1fe7/68747470733a2f2f6c75636964652e6465762f6170692f67682d69636f6e2f50484e325a776f67494868746247357a50534a6f644852774f693876643364334c6e637a4c6d39795a7938794d4441774c334e325a79494b4943423361575230614430694d6a516943694167614756705a326830505349794e43494b4943423261575633516d393450534977494441674d6a51674d6a5169436941675a6d6c7362443069626d39755a53494b4943427a64484a7661325539496d4e31636e4a6c626e5244623278766369494b4943427a64484a766132557464326c6b64476739496a496943694167633352796232746c4c577870626d566a59584139496e4a766457356b49676f6749484e30636d39725a5331736157356c616d3970626a3069636d3931626d5169436a344b49434138634746306143426b50534a4e4d5463674d326730646a51694943382b4369416750484268644767675a443069545445344c6a55334e5341784d5334774f444a684d544d674d544d674d434177494445674d5334774e4467674f5334774d6a63674d5334784e7941784c6a4533494441674d4341784c5445754f5445304c6a55354e3077784e4341784e7949674c7a344b49434138634746306143426b50534a4e4e7941784d43417a4c6a4935494459754d6a6c684d5334784e7941784c6a4533494441674d434178494334324c5445754f5445674d544d674d544d674d434177494445674f5334774d7941784c6a41314969417650676f67494478775958526f49475139496b303349444530595445754e7941784c6a63674d434177494441744d5334794d4463754e5777744d6934324e4459674d6934324e445a424c6a55754e534177494441674d43417a4c6a55674d5468494e574578494445674d434177494445674d534178646a45754e5745754e533431494441674d434177494334344e5451754d7a553054446b754e5341784f4334794d4464424d533433494445754e794177494441674d4341784d4341784e3359744d6d4578494445674d434177494441744d533078656949674c7a344b49434138634746306143426b50534a4e4f5334334d4463674d5451754d6a6b7a4944497849444d694943382b436a777663335a6e50673d3d2e737667">Open lucide studio](https://lucide-studio.vercel.app/edit?value=%3Cpath+d=%22M17+3h4v4%22+/%3E%3Cpath+d=%22M18.575+11.082a13+13+0+0+1+1.048+9.027+1.17+1.17+0+0+1-1.914.597L14+17%22+/%3E%3Cpath+d=%22M7+10+3.29+6.29a1.17+1.17+0+0+1+.6-1.91+13+13+0+0+1+9.03+1.05%22+/%3E%3Cpath+d=%22M7+14a1.7+1.7+0+0+0-1.207.5l-2.646+2.646A.5.5+0+0+0+3.5+18H5a1+1+0+0+1+1+1v1.5a.5.5+0+0+0+.854.354L9.5+18.207A1.7+1.7+0+0+0+10+17v-2a1+1+0+0+0-1-1z%22+/%3E%3Cpath+d=%22M9.707+14.293+21+3%22+/%3E)

Yeah, that looks nice.